### PR TITLE
dotnet: emit namespace/class features for ldvirtftn/ldftn instructions

### LINKED ISSUE #1117: dotnet: emit features for ldftn/ldvirftn instructions
- `ldftn`: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.opcodes.ldftn?view=net-6.0
- `ldvirftn`: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.emit.opcodes.ldvirtftn?view=net-6.0

e.g.

```
... ldftn     instance void malware.methods::timer_callback(object)
... newobj    instance void [mscorlib]System.Threading.TimerCallback::.ctor(object, native int)
```

emit

```yaml
- namespace: malware
- class: malware.methods
- function-name: malware.methods::timer_callback
```

### PR DESCRIPTION
**Pull request by mike-hunhoff**
closes #1117

### CODE REVIEW

File: capa/features/extractors/dnfile/extractor.py [hunk 1]
```diff
@@ -57,17 +57,24 @@ def get_functions(self) -> Iterator[FunctionHandle]:
         # calculate unique calls to/from each method
         for fh in methods.values():
             for insn in fh.inner.instructions:
-                if insn.opcode not in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli, OpCodes.Newobj):
+                if insn.opcode not in (
+                    OpCodes.Call,
+                    OpCodes.Callvirt,
+                    OpCodes.Jmp,
+                    OpCodes.Newobj,
+                ):
```
Comment (mike-hunhoff) [inline capa/features/extractors/dnfile/extractor.py:65]: tighten types (see ECMA VI.C.4.8)

File: capa/features/extractors/dnfile/insn.py [hunk 2]
```diff
@@ -96,7 +96,12 @@ def get_callee(ctx: Dict, token: Token) -> Union[DnType, DnUnmanagedMethod, None
 
 def extract_insn_api_features(fh: FunctionHandle, bh, ih: InsnHandle) -> Iterator[Tuple[Feature, Address]]:
     """parse instruction API features"""
-    if ih.inner.opcode not in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli, OpCodes.Newobj):
+    if ih.inner.opcode not in (
+        OpCodes.Call,
+        OpCodes.Callvirt,
+        OpCodes.Jmp,
+        OpCodes.Newobj,
+    ):
```
Comment (mike-hunhoff) [inline capa/features/extractors/dnfile/insn.py:104]: tighten types (see ECMA VI.C.4.8)

File: capa/features/extractors/dnfile/insn.py [hunk 3]
```diff
@@ -116,7 +121,7 @@ def extract_insn_property_features(fh: FunctionHandle, bh, ih: InsnHandle) -> It
     name: Optional[str] = None
     access: Optional[str] = None
 
-    if ih.inner.opcode in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp, OpCodes.Calli):
+    if ih.inner.opcode in (OpCodes.Call, OpCodes.Callvirt, OpCodes.Jmp):
```
Comment (mike-hunhoff) [inline capa/features/extractors/dnfile/insn.py:124]: tighten types (see ECMA VI.C.4.8)

**Review by mr-tz — APPROVED** [review]
Nice, any chance we can add a test case from the existing binaries?

**Pull request merged**